client/components/jetpack/connection-health/test/index.jsx

```diff
@@ -3,7 +3,7 @@
  */
 import { screen } from '@testing-library/react';
 import { merge } from 'lodash';
-import { jetpackConnectionHealth } from 'calypso/state/jetpack-connection-health/reducer';
+import { reducer as jetpackConnectionHealth } from 'calypso/state/jetpack-connection-health/reducer';
```

Comment:
I wanted the full reducer here, without the `storage` part, and this is why I imported it like that, so this excludes the second option.
Regarding the first one, this would import only the `connectionHealth` reducer part, which for now is fine ( as I didn't add more tests regarding the other parts of the reducer ), but as those tests supposed to test the whole reducer object, I'll leave it as is! Thanks for pointing this out!